docs/admin/whiteboard.rst

```diff
@@ -1,40 +1,39 @@
 Whiteboard messages
 ===================
 
-You can use whiteboard messages to give some information to your translators.
-The message can be site-wide or targeted to a translation component or language.
+Provide infor to your translators.by posting messages, site-wide, per component or language.
 
-This can be useful for various things from announcing the purpose of the website to
-specifying targets for translations.
+Announce the purpose, deadlines, status, or specify targets for translations.
 
-The whiteboard can currently be specified only in the admin interface:
+Adding whiteboard messages can be done from the admin interface, or each respective cateogry for admins:
 
 .. image:: /images/whiteboard.png
 
-The whiteboard messages are then shown based on specified context:
+The whiteboard messages are then shown based on their specified context:
 
 No context specified
 
     Shown on dashboard (landing page).
 
 Project specified
 
-    Shown on project, all its components and translations.
+    Shown for project, and all its components and translations.
 
 Component specified
 
-    Shown on component and all its translations.
+    Shown for a component and all its translations.
 
 Language specified
 
-    Shown on language overview and all translations in this language.
+    Shown on the language overview and all translations in that language.
 
 
-You can see how it looks on the language overview page:
+This is how it looks on the language overview page:
 
 .. image:: /images/whiteboard-language.png
-
+   alt : Image showing a message that reads: "Czech translators rock!" atop the Czech language overview.
 
 And on the project page:
 
 .. image:: /images/whiteboard-project.png
+   alt : Image showing a message that reads: "Translations will be used only if they reach 60%" atop the dashboard view.
```

Comment:
This should be discouraged. Can't think of something to replace it right now. (Needs the image too).